tests/test_helpscout_all_fields.py

```diff
@@ -0,0 +1,72 @@
+from tap_tester import menagerie, connections, runner
+
+from base import HelpscoutBaseTest
+
+class AllFieldsTest(HelpscoutBaseTest):
+
+    def name(self):
+        return "tap_helpscout_tests_using_shared_token_chaining"
+
+    def test_name(self):
+        print("Pagination Test for tap-helpscout")
```

Comment:
The name here says pagination.

Reply:
oh i forgot about this and the change is merged to master, can i make this change in my next branch and push it ?